asv_bench/benchmarks/io/csv.py

```diff
@@ -555,4 +555,19 @@ def time_read_csv_index_col(self):
         read_csv(self.StringIO_input, index_col="a")
 
 
+class ReadCSVDatePyarrowEngine(StringIORewind):
```

Comment:
Can you fuse this benchmark with ``ReadCSVParseDates`` above?
(You might have to disable ``time_multiple_dates`` for the pyarrow engine)

Reply:
I'd prefer keeping them separate for now since I am expecting to add more arrow versions that aren't really compatible with the others.